presto-docs/src/main/sphinx/connector/sqlserver.rst

```diff
@@ -7,6 +7,8 @@ external `Microsoft SQL Server <https://www.microsoft.com/sql-server/>`_ databas
 different systems like SQL Server and Hive, or between two different
 SQL Server instances.
 
+It has been tested against SQL Server 2017-CU21 and 2019-CU6.
```

Comment:
We are using 2017-CU13 (≠2017-CU21) in product test. 
https://github.com/prestosql/presto/blob/1b4275037b6f834dcb230d9ec9e6f028564216b5/presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/environment/SinglenodeSqlserver.java#L65

Reply:
Missed the product tests -- Thanks. I'll update that as well.

Reply:
Updated comment to be more clear of my intentions.  2017 is supported but currently tested against 2019.